server/parameters/CoreConfigParser.java

```diff
@@ -47,21 +47,21 @@
 import static com.vaticle.typedb.core.server.parameters.util.YamlParser.predefined;
 import static com.vaticle.typedb.core.server.parameters.util.YamlParser.restricted;
 
-public class ConfigParser extends YamlParser.Value.Compound<Config> {
+public class CoreConfigParser extends YamlParser.Value.Compound<CoreConfig> {
```

Comment:
There are three kinds of changes in this class:

1. Prefix `Config` and `ConfigParser` with `Core`
2. Change access modifier of fields from `private` to `protected` for extensibility from its subclasses.
3. Change access modifier of fields from `private` to `public` for extensibility, for fields that are intended for reuse in other classes, not just subclasses.